__tests__/types/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "jsx": "preserve",
```

Comment:
This is not needed for the PR, but it removes annoying errors from test files seen in vscode